sympy/physics/control/lti.py

```diff
@@ -24,19 +34,48 @@ def var(self):
         return self._var
 
     def __add__(self, other):
-        pass
+        if not self.var == other.var:
+            raise ValueError("Last argument for both Transfer Functions must be same.")
+        p = self.num * other.den + other.num * self.den
+        q = self.den * other.den
+        return TransferFunction(expand(p), expand(q), self.var)
```

Comment:
Again, why are we expanding by default? 
I think that should be left to the user.

Reply:
Same in all the other functions

Reply:
Yes, we should try to avoid using `expand` in library code but consider this:
``` python
>>> tf1 = TransferFunction(p*s**4 + 3*s**3 + 2*s**2 - 5*p + 7, s**3 - 6*s**2 + p - s, s)
>>> tf2 = TransferFunction(s**5 + s**3 - s**2 + s - 5, 4*s**3 - 2*s**2 + 10, s)
>>> tf1 + tf2
TransferFunction((4*s**3 - 2*s**2 + 10)*(p*s**4 - 5*p + 3*s**3 + 2*s**2 + 7) + (p + s**3 - 6*s**2 - s)*(s**5 + s**3 - s**2 + s - 5), (4*s**3 - 2*s**2 + 10)*(p + s**3 - 6*s**2 - s), s)
```
Leaving it as it is might look good when `num` and `den` are as simple as `(s - 4)*(s + 3)` or not higher degree polynomials.. but while dealing with degree 4 or more, I think using `expand` would be a better decision. Python-control also expands `num` and the `den`. Should I remove `expand` from all of them?

Reply:
Yes, it might not look that good. But usability is more important. The users might want to use unexpanded equations. 
These large polynomials will look bad anyway after a couple more operations. So, it will be better to leave it as it is and let the users expand explicitly if they want. It's just one more step.

Reply:
> So, it will be better to leave it as it is and let the users expand explicitly if they want

`expand` doesn't work for the `TransferFunction` which is strange cause other functions like `factor` & `xreplace` are working fine. Same case with `simplify`, it doesn't work! It might work if we were using `Mul` to represent TF, so I am using this version of `simplify`:

``` python
    def simplify(self):
        tf = cancel(Mul(self.num, 1/self.den)).as_numer_denom()
        num_, den_ = tf[0], tf[1]
        return TransferFunction(num_, den_, self.var)
```